cmd/microcloud/preseed.go

```diff
@@ -159,11 +230,30 @@ func (c *initConfig) RunPreseed(cmd *cobra.Command) error {
 		return err
 	}
 
+	initiator := config.isInitiator(c.name, c.address)
+
+	fmt.Println("initiator", initiator)
+	fmt.Println("bootstrap", c.bootstrap)
+
+	if !ready && !c.bootstrap && initiator {
+		return fmt.Errorf("MicroCloud isn't yet initialized, run 'microcloud preseed' instead")
```

Comment:
This error's a bit odd, since there's only one command now. Maybe just mention that `c.name` or `c.address` cannot be the initiator?

Reply:
Yeah I think I have removed the `--add` flag but left the message.